src/scripts/OSFramework/OSUI/Pattern/SectionIndex/scss/_sectionindex.scss

```diff
@@ -72,6 +72,22 @@
 	}
 }
 
+// OS HighContrast Enabled -------------------------------------------------------
+///
+.os-hight-contrast {
+	.osui-section-index::before {
+		border-left: 1px solid var(--color-focus-outer);
+	}
+
+	.osui-section-index-item {
+		&--is-active {
+			&::before {
+				border-left: 4px solid var(--color-focus-outer);
```

Comment:
4px doesn't exist on OSUI design system border values. Could we use var(--border-size-l) with 3px?

Reply:
Well, it will not have that much contrast but in consistency favor I'll use the `--boder-size-l`!
Done!